pages/common/adb-shell.md

```diff
@@ -0,0 +1,32 @@
+# adb shell
+
+> Android Debug Bridge Shell: Run remote shell command with an Android emulator instance or connected Android devices.
+> More information: <https://developer.android.com/studio/command-line/adb>.
+
+- Start a remote interactive shell in the emulator/device:
+
+`adb shell`
+
+- Reset dangerous permissions to an application:
+
+`adb shell pm reset-permissions {{package}}`
+
+- Revoke a dangerous permission to an application:
+
+`adb shell pm revoke {{package}} {{permission}}`
+
+- Trigger a key event:
+
+`adb shell input keyevent {{keycode}}`
+
+- Clear the data from an emulator/device's application:
+
+`adb shell pm clear {{package}}`
+
+- Start an activity on emulator/device:
+
+`adb shell am start -n {{package}}/{{activity}}`
+
+- Start home activity on emulator/device:
```

Comment:
Perhaps this will read better?
```suggestion
- Start the home activity on an emulator or device:
```

Also, is this the main home screen, or the home screen of a specific application? If it's the latter, then perhaps something like this would be better?
```suggestion
- Start the home activity for an application on an emulator or device:
```

Finally, are there any long forms of the arguments used here? We generally prefer long forms of arguments in tldr pages.

Reply:
Is the Android home.

adb has only short forms for most arguments